vertica/datadog_checks/vertica/vertica.py

```diff
@@ -653,3 +575,8 @@ def parse_metric_groups(self):
         self._metric_groups.extend(
             default_metric_groups[group] for group in default_metric_groups if group in metric_groups
         )
+
+    def _maybe_gauge(self, name, value, *args, **kwargs):
+        "Record a gauge metric only if the value is not None"
+        if value is not None:
+            self.gauge(name, value, *args, **kwargs)
```

Comment:
I don't think we need this https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/checks/base.py#L654